tests/test_loss_masking.py

```diff
@@ -21,6 +21,11 @@ def test_masking():
     loss = model.train_on_batch(x, y)
     assert loss == 0
 
+    x = np.array([[[0], [0]],
```

Comment:
Could it be a separate test? Tests should be as small as possible and modular. You should make a helper function to create the model and avoid code duplication.

something like `test_mask_is_all_zeros()`.

Reply:
@gabrieldemarmiesse I've updated the tests.